spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/controller/StreamDefinitionController.java

```diff
@@ -264,7 +276,11 @@ public StreamDefinitionResource save(@RequestParam("name") String name, @Request
 		}
 		this.repository.save(stream);
 		if (deploy) {
-			deploymentController.deploy(name, null);
+			Map<String, String> streamDeploymentProperties = new HashMap<>();
+			if (useSkipper) {
```

Comment:
I placed the properties to use skipper outside the `app` and `deployer` prefix since it is a larger 'switch'.  For one thing, one wouldn't use skipper to deploy some of the apps in the stream and the AppDeployer instance in SCDF to deploy the rest.   There is also a bit of overloading going on with the term 'deployer' now.  The prefix deployer is what should be passed into the AppDeployer instance.  `StreamDeployer` is a new top level 'deployer' as well.  maybe it should be a prefix `streamDeployer.useSkipper=true`.  ?